xml/System.Text.Json/JsonElement.xml

```diff
@@ -777,9 +789,8 @@ This method does not parse the contents of a JSON string value.
         <summary>Gets the value at a specified index when the current value is an <see cref="F:System.Text.Json.JsonValueKind.Array" />.</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
-        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueKind.Array" />.</exception>
-        <exception cref="T:System.IndexOutOfRangeException">
-          <paramref name="index" /> is not in the range [0, <see cref="M:System.Text.Json.JsonElement.GetArrayLength" />()).</exception>
+        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.ValueKind" /> is not <see cref="F:System.Text.Json.JsonValueKind.Array" />.</exception>
```

Comment:
We are missing parameter description of this indexer:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.item?view=netcore-3.0#System_Text_Json_JsonElement_Item_System_Int32_

https://github.com/dotnet/dotnet-api-docs/blob/c7324dff76601c7fdfeb798358fe63a43e61db3a/xml/System.Text.Json/JsonElement.xml#L787-L795

It doesn't exist in source either.

@bartonjs, can you please wordsmith something here to describe the `JsonValueKind.Array` indexer?
https://github.com/dotnet/corefx/blob/9e994a5923d426f0e64ed3b8f9b93df6f214a42a/src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs#L40-L61